.github/workflows/security.yml

```
@@ -0,0 +1,112 @@
+  pull_request:
+    branches: [main, master]
+  schedule:
+    - cron: '0 2 * * *'  # Daily at 2 AM UTC
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  security-events: write
+  actions: read
+
+env:
```

actionlint (1.7.11)

[error] expecting a single ${{...}} expression or mapping value for "env" section, but found plain text node

(syntax-check)